src/lib/transformSpec.ts

```
@@ -0,0 +1,20 @@
+export function transformSpec(spec) {
+  console.warn('transformSpec', spec)
+  if (!spec) {
+    return
+  }
+
+  if (!spec.openapi || !spec.openapi.startsWith('3.')) {
+    throw new Error('Only OpenAPI 3.x is supported')
+  }
+
+  if (spec?.paths) {
+    spec = generateMissingOperationIds(spec)
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)